controllers/discovery/initDiscoveryUserData.js

```diff
@@ -112,6 +118,7 @@ const InitDiscoveryUserData = async (req, res) => {
           AND A.user_id != :admin_user_id
           ${where_anon_dm} 
           ${filterBlockedUser}
+          ${filterExcludedUser}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The inclusion of `filterExcludedUser` in the SQL query is done correctly.
```diff
- ${filterBlockedUser}
+ ${filterBlockedUser}
+ ${filterExcludedUser}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->